src/stylesheets/packages/_uswds-components.scss

```diff
@@ -14,10 +14,10 @@
 @import "../elements/layout-grid";
 @import "../elements/table";
 @import "../elements/tags";
-@import "../elements/typography/content";
 @import "../elements/typography/links";
 @import "../elements/typography/list";
 @import "../elements/typography/prose";
+@import "../elements/typography/content";
```

Comment:
Why did this need to move in the import order?

Reply:
If links are removed from usa-prose you wouldn't need to. When they were displaying there was a [color a11y issue](https://app.circleci.com/pipelines/github/uswds/uswds/2047/workflows/b615ec35-0bd8-412f-b54c-5ddfc3507abf/jobs/6431) for the link on dark background, bc of a specificity issue.